ON-CALL.md

```diff
@@ -0,0 +1,43 @@
+# Istio On-call Playbook
+
+## Who
+Every week, one Istio developer is on-call and is responsible for maintaining Istio build and help out users and other developers. The on-call person should prioritize on-call duties on top of their daily work.
+
+## Schedule
+The on-call schedule can be found [here][https://docs.google.com/spreadsheets/d/1FaHwPpad3F3hva2suJweNeTnocjKtLnbgLkyMRPzgUY/edit#gid=1475801904].
```

Comment:
Incorrect md syntax.
Use () after [] for hyperlinks.

You can preview the text using the "view" button above too.

Reply:
The [] works for me in my golang IDE viewer, but I will change it.